Rectangle/WindowCalculation/MoveLeftRightCalculation.swift

```diff
@@ -8,77 +8,98 @@
 
 import Cocoa
 
-class MoveLeftRightCalculation: WindowCalculation {
+// Applicable options:
+// Defaults.subsequentExecutionMode.traversesDisplays (only applies when resizeOnDirectionalMove is disabled)
+// Defaults.centeredDirectionalMove.enabled (independent)
+// Defaults.resizeOnDirectionalMove.enabled
+
+class MoveLeftRightCalculation: WindowCalculation, RepeatedExecutionsCalculation {
     
     override func calculate(_ params: WindowCalculationParameters) -> WindowCalculationResult? {
-        let usableScreens = params.usableScreens
-        if params.action == .moveLeft {
-            return calculateLeft(params, screen: usableScreens.currentScreen)
-        } else if params.action == .moveRight {
-            return calculateRight(params, screen: usableScreens.currentScreen)
+        
+        if Defaults.resizeOnDirectionalMove.enabled {
```

Comment:
This will ignore `Defaults.subsequentExecutionMode.traversesDisplays` when `resizeOnDirectionalMove` is enabled. I think I would prefer to have it still honor the `traversesDisplays` setting since that toggle is currently in the UI.

Reply:
I’ve updated this behavior and the docs to reflect this.